app/assets/javascripts/admin/addon/components/schema-theme-setting/editor.gjs

```diff
@@ -317,16 +324,14 @@ export default class SchemaThemeSettingEditor extends Component {
             </li>
 
             {{#each node.trees as |nestedTree|}}
-              {{#if (gt nestedTree.nodes.length 0)}}
```

Comment:
This conditional needs to be removed because even if there are no nodes, we want to display the heading as the button to "add node" will be present.